GiniVision/Classes/Core/Extensions/Collection.swift

```diff
@@ -8,10 +8,34 @@
 
 import Foundation
 
+extension Collection {
+    var isNotEmpty: Bool {
+        return !isEmpty
+    }
+}
+
 internal extension Collection where Iterator.Element == CFString {
     
     var strings: [ String ] {
         return self.map { $0 as String }
     }
     
 }
+
+extension Collection where Iterator.Element == GiniVisionDocument {
+    var isAssorted: Bool {
```

Comment:
So Assorted means that all elements have the same document type?
How about calling it a "homogenous document type"?
Also, can't you make it simpler? Why not get the type of the first one and then filter/reduce to the ones who don't have the same type. If the array is not empty, the collection is "assorted".

Reply:
I didn't like neither _Mixed_ nor _Diversed_, so I decided to go with _Assorted_. But how would you call that long `homogenous document type` variable? That is what I wanted to express but IMO is clear with one word. What do you think?

Reply:
Like that way of getting the bool btw